src/apps/common/submodules/appSelector/appSelector.scss

```
@@ -179,38 +183,82 @@
 			overflow-y: auto;
 			border-top: solid 1px $french-gray2;
 
-			.app-list {
+			ul.app-list {
 				margin-left: 0;
 
-				.app-item {
+				li {
 					box-sizing: border-box;
 					width: 100%;
-					background-color: $panel-background-color;
-					box-shadow: inset 0px 1px 0px 0px $french-gray2;
-					cursor: pointer;
-					display: flex;
-					flex-wrap: wrap;
-
-					&:hover {
-						background-color: $background-primary-color;
-					}
 
-					&.selected {
-						background-color: $french-pass;
+					&.app-item {
+						background-color: $panel-background-color;
+						cursor: pointer;
+						display: flex;
+						flex-wrap: wrap;
+						box-shadow: inset 0px 1px 0px 0px $white,
+							inset 0px -1px 0px 0px $white;
+
+						&:hover, &.selected {
+							background-color: $french-pass;
+						}
+
+						&.even:not(.selected) .text-container {
+							background-color: $athens-gray;
+
+							&:hover {
+								background-color: transparent;
+								transition: none;
+							}
+						}
+
+						.text-container {
+							width: 100%;
+							transition: background-color 250ms linear;
+
+							&:hover {
+								background-color: transparent;
+								transition: none;
+							}
+						}
+
+						.app-icon {
+							flex: initial;
+							margin-right: 0.75rem;
+							margin-bottom: 0.75rem;
+						}
+						.app-title {
+							flex: 1;
+							margin-bottom: 0.75rem;
+							align-self: center;
+						}
+						.app-description {
+							width: 100%;
```

App descriptions are actually truncated to only appear on a single line. If I had to guess, I'd say the `max-width` is about 75%.

To achieve the ellipsis effect, you can combine the following properties:
```suggestion
							width: 75%;
							white-space: nowrap;
							overflow-x: hidden;
							text-overflow: ellipsis;
```

---

Thanks, I'll truncate the descriptions this way.